util/test/paratest.chapcs

```diff
@@ -133,17 +134,27 @@ def get_good_nodepara():
     return nodepara
 
 
-def run_paratest(args):
+def run_paratest(paratest_args, nightly=False):
     """
     Run paratest inside an salloc using all nodes that are not reserved
     exclusively on the chapel partition. Throw `--oversubscribe --nice` and
     turn off affinity and limit how many executables can run at once so we play
     nice with other testing going on.
     """
-    nodepara = get_good_nodepara()
-    num_free_nodes = get_num_non_exclusive_nodes()
-    para_env = ['-env', 'CHPL_TEST_LIMIT_RUNNING_EXECUTABLES=yes', '-env',
-                'CHPL_RT_OVERSUBSCRIBED=yes']
+    if not debug:
```

Comment:
Can we rename `debug` to `dry_run` or `debug_no_slurm` or something and pull it into a different PR? Personally, I  think the debug branches make the script harder to follow and most of the script is slurm commands so I don't mind debugging/running on chapcs, but if it was useful for debugging I don't mind keeping it.

Reply:
Yeah I see your point. I like testing things as much as possible locally, though I don't know whether I think strongly enough to create another PR for that.

I'll remove this stuff